.github/workflows/pr.yaml

```diff
@@ -17,11 +17,28 @@ jobs:
       - conda-python-tests
       - wheel-build
       - wheel-tests
+      - telemetry-setup
     secrets: inherit
-    uses: rapidsai/shared-workflows/.github/workflows/pr-builder.yaml@branch-25.04
+    uses: rapidsai/shared-workflows/.github/workflows/pr-builder.yam@branch-25.04
```

Comment:
```suggestion
    uses: rapidsai/shared-workflows/.github/workflows/pr-builder.yaml@branch-25.04
```

Reply:
Feeling potato today. Maybe sweet.

Reply:
I've had sweet potato fries 3 of the last 4 calendar days. I am a beacon of health. Be like me.